### PR TITLE
Support pfx passphrase

### PR DESCRIPTION
**Pull request by spillevink**

### CODE REVIEW

File: lib/server/utils.js [hunk 1]
```diff
@@ -21,11 +21,13 @@ var utils = {
     },
     /**
      * @param filePath
+     * @param passphrase
      * @returns {{pfx}}
      */
-    getPFX: function (filePath) {
+    getPFX: function (filePath, passphrase) {
```
Comment (shakyShane) [inline lib/server/utils.js]: How would this work if `passphrase` here is undefined?

Comment (spillevink) [inline lib/server/utils.js]: `https.createServer` does not care about undefined arguments.

Comment (shakyShane) [inline lib/server/utils.js]: so there's no risk here?

Comment (spillevink) [inline lib/server/utils.js]: I don't think so, and the PFX tests with no passphrase still seem to be passing.

**Comment by shakyShane** [comment]
sorry for the delay @spillevink - this will go up in the next release, thanks :)

**Pull request merged**